### ISSUE TITLE
AMD/Requirejs compatiblity

### ISSUE THREAD

**Issue by nitinsurana** [nathancahill/Split.js#31]
Creates a global function, please implement a way to prevent global namespace pollution and export the module, in short AMD support.

**Comment by nitinsurana** [comment]
The source-code seems to have it, will close it for now. Sorry for the trouble.

**Comment by nitinsurana** [comment]
require(['split'],function(Split){ Split(...)});

throws exception saying Split is not a function.

**Comment by fsuter** [comment]
I'm having the same issue. Any help would be very appreciated.

**Comment by nathancahill** [comment]
I'll take a look at this.

**Comment by fsuter** [comment]
Great, thanks in advance.

If that helps, here's the code I tried (and maybe I'm simply wrong; I'm not yet entirely familiar with RequireJS):

```javascript
define('TYPO3/CMS/Templatedisplay/MappingInterface', ['jquery, ./split.min'], function($, Split) {
    'use strict';

    var MappingInterface = {};

    TYPO3.jQuery(function() {
        Split(['#field-mapping', 'field-configuration'], {
            sizes: [60, 40],
            minSize: 200
        });
    });

    return MappingInterface;
});
```

**Comment by nathancahill** [comment]
Fixed in 1.2.0+
